klient/k8s/yaml/yaml.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package yaml
```

Comment:
```suggestion
package ???
```

Is this better off in `sigs.k8s.io/e2e-framework/klient/k8s/resources` somehow?

Reply:
Since this package takes care of loading both yaml and json, might be better to rename this as `loader` and create it under `sigs.k8s.io/e2e-framework/klient/k8s/resources` ?